docs/MODEL_UPLOAD.md

```diff
@@ -0,0 +1,33 @@
+# How to bring a new Deep Learning model on Disco : [DIS]tributed [CO]llaborative Learning
+
+You must first bring your model to a TensorFlow JS format, consisting of a TensorFlow.js model file in a JSON format, and a weight file in .bin format.
+
+We distinguish two use cases :
+
+## I am a developper who wants to define my own task
+
+For more information on the task object instanciation itself, please refer to the TASK.md file in our documentation.
+
+ - In ```disco/discojs/discojs-core/src/tasks/``` define your new custom task by instanciating a Task object, and define the async function ```model```. You will need to have your model in the .json + .bin format.
+ - In ```disco/discojs/discojs-core/src/tasks/index.ts``` export your newly defined task
+ - Run the ```./build.sh``` script from ```disco/discojs/discojs-core```
+ - Reinstall cleanly the server by running ```npm ci``` from ```disco/server```
+ - Reinstall cleanly the client by running ```npm ci``` from ```disco/web-client```
+ - Instantiate a Disco server by running ```npm run dev``` from ```disco/server```
+ - Instanciate a Disco client by running ```npm run dev``` from ```disco/web-client```
+ Your task has been successfully uploaded.
+ 
+## I am a user who wants to define my custom task and upload my model to Disco
```

Comment:
maybe we can mention the fact that the task will be defined on _our_ disco server?